src/backend/Csrs.Api/Health/HealthCheckExtensions.cs

```diff
@@ -1,43 +1,43 @@
-﻿using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+﻿using Csrs.Api.Health;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
 using Microsoft.Extensions.Diagnostics.HealthChecks;
 
-namespace Csrs.Api.Health
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
Changed namespace so that in Program.cs, we do not require additional using statements and the `AddXXX` method is just available.